cmd/prototype.go

```diff
@@ -92,7 +92,23 @@ var prototypeListCmd = &cobra.Command{
 			return fmt.Errorf("Command 'prototype list' does not take any arguments")
 		}
 
-		index := prototype.NewIndex([]*prototype.SpecificationSchema{})
+		cwd, err := os.Getwd()
+		if err != nil {
+			return err
+		}
+		wd := metadata.AbsPath(cwd)
+
+		manager, err := metadata.Find(wd)
```

Comment:
We should probably open a ticket to separate the prototype logic into the `kubecfg` package so we don't have to have the redundant code in getting `manager`.

Reply:
Prototype code probably can't go into the kubecfg package, since that's going to be a shared package. But, can you tell me more about which code you see as being rendundant?

Reply:
Hm, I've always thought of `kubecfg` as a package for ksonnet that handles more internal command logic outside of command flag parsing done by `cmd`.

I'm referring to the common code that gets the working directory, and then finds the manager. I added a shared function `manager()` in the `kubecfg` package that does that in an earlier commit.

Reply:
Ok, we can do that. https://github.com/ksonnet/ksonnet/issues/65